2.4/Dockerfile.rhel7

```diff
@@ -16,6 +16,13 @@ ENV MONGODB_VERSION         2.4
 ENV IMAGE_DESCRIPTION       MongoDB 2.4
 ENV IMAGE_TAGS              mongodb,mongodb24
 ENV IMAGE_EXPOSE_SERVICES   27017:mongodb
+ENV container docker
+
+LABEL RUN docker run -d -e MONGODB_USERNAME=mongodb-user -e MONGODB_PASSWORD=pa55word -e MONGODB_ADMIN_PASSWORD=admpa55word -e MONGODB_DATABASE=testing -v ${DATADIR}:/var/lib/mongodb/data IMAGE 
```

Comment:
Wont the `LABEL` instruction break our builds ? Since they are not running on Docker 1.6.0